cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/model/RandomCluster.java

```diff
@@ -47,8 +47,13 @@ public static ClusterModel generate(Map<ClusterProperty, Number> clusterProperti
     int numRacks = clusterProperties.get(ClusterProperty.NUM_RACKS).intValue();
     int numBrokers = clusterProperties.get(ClusterProperty.NUM_BROKERS).intValue();
     BrokerCapacityConfigFileResolver configFileResolver = new BrokerCapacityConfigFileResolver();
-    configFileResolver.configure(Collections.singletonMap(BrokerCapacityConfigFileResolver.CAPACITY_CONFIG_FILE,
-                                                          RandomCluster.class.getClassLoader().getResource("DefaultCapacityConfig.json").getFile()));
+    if (clusterProperties.get(ClusterProperty.POPULATE_DISKS).intValue() > 0) {
+      configFileResolver.configure(Collections.singletonMap(BrokerCapacityConfigFileResolver.CAPACITY_CONFIG_FILE,
+                                   RandomCluster.class.getClassLoader().getResource("testCapacityConfigJBOD.json").getFile()));
```

Comment:
can we avoid hardcoding `testCapacityConfigJBOD.json` and `DefaultCapacityConfig.json`?